CMakeLists.txt

```diff
@@ -96,7 +96,7 @@ endif ()  # TRITON_ENABLE_GPU
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-unused-variable -Wno-unused-function")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-unused-variable -Wno-unused-function -Wno-unused-local-typedefs")
```

Comment:
OK, since it unblocks CI - but in the long run, shouldn't we rather treat DALI as a system library? This would suppress all warnings in DALI headers.

Reply:
I'm not sure how feasible it is in the current setup.